pages/common/goimports.md

```diff
@@ -0,0 +1,12 @@
+# goimports
+
+> Updates your Go import lines, adding missing ones and removing unreferenced ones.
+> More information: https://godoc.org/golang.org/x/tools/cmd/goimports.
+
+- Display diffs instead of rewriting file:
+
+`goimports -d main.go`
+
+- Write result to (source) file instead of stdout:
+
+`goimports -w main.go`
```

Comment:
I think you should use token syntax as describe in below link instead of 'main.go' here.

https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md#token-syntax